projects/aca-about/src/lib/about.component.scss

```diff
@@ -2,3 +2,13 @@ adf-about {
   padding: 10px;
   width: 100%;
 }
+
+.app-about-container {
+  position: fixed;
```

Comment:
this is incorrect, we should not require fixed positioning for proper layouting; width and height to 100% should do the trick